### PR TITLE
[OSDOCS#9796] Add note about ShiftStack CPMS gotchas to release notes

### PR DESCRIPTION
**Pull request by maxwelldb**
Needs change management. See https://issues.redhat.com/browse/OSDOCS-9796 for info.

Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-9796

Link to docs preview: https://72319--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-mapi-cpms-shiftstack-support

QE review: Not needed. This is bubbling already reviewed information up into the features section of the notes.
- [ ] ~~QE has approved this change.~~

Additional information: ~~Needs change management.~~ CM done.

### CODE REVIEW

**Comment by kalexand-rh** [comment]
LGTM

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1077,6 +1077,13 @@ With this release, control plane machine sets are supported for clusters that ru
 
 For more information, see xref:../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-getting-started[Getting started with the Control Plane Machine Set Operator].
 
+[NOTE]
+====
+For clusters that have root volume availability zones and are running on {rh-openstack} that you upgrade to {product-version} and have root volume availability zones, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in the related link:https://access.redhat.com/solutions/7013893[knowledge base article].
```
Comment (bergerhoffer) [inline release_notes/ocp-4-14-release-notes.adoc:1082]: It might be unavoidable, but the first sentence is very long and hard to follow. Is it possible to break it up a bit more? Or perhaps is the second instance of "and have root volume availability zones" not required?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1077,6 +1077,13 @@ With this release, control plane machine sets are supported for clusters that ru
...
+
+For clusters that have compute zones configured with at least one zone and are running on {rh-openstack}, which is upgradable to version {product-version}, root volumes must now also be configured with at least one zone. If this configuration change does not occur, a control plane machine set cannot be generated for your cluster. To make the required change, follow the instructions in the related link:https://access.redhat.com/solutions/7024383[knowledge base article].
```
Comment (bergerhoffer) [inline release_notes/ocp-4-14-release-notes.adoc:1084]: Since these are two different kbase article links, it might be helpful to users to list the titles of the articles, instead of just "knowledge base article" as the link text.

See https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links for more info on linking to kbase articles

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -2959,9 +2966,9 @@ To revert all nodes in the cluster to the cgroups v2 configuration, you must edi
...
 
-* For clusters that have root volume availability zones and are running on {rh-openstack} that you upgrade to {product-version} and have root volume availability zones, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in the link:https://access.redhat.com/solutions/7013893[Knowledgebase article]. (link:https://issues.redhat.com/browse/OCPBUGS-13300[*OCPBUGS-13300*])
+* For clusters that have root volume availability zones and are running on {rh-openstack} that you upgrade to {product-version} and have root volume availability zones, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in the link:https://access.redhat.com/solutions/7013893[knowledge base article]. (link:https://issues.redhat.com/browse/OCPBUGS-13300[*OCPBUGS-13300*])
```
Comment (bergerhoffer) [inline release_notes/ocp-4-14-release-notes.adoc:2969]: We don't write ours as two words

See https://redhat-documentation.github.io/supplementary-style-guide/#knowledgebase and https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links

**Review by bergerhoffer — COMMENTED** [review]
A few suggestions!

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (maxwelldb) [reply release_notes/ocp-4-14-release-notes.adoc:1082]: Trying to keep things the same as the known issues later on, but yeah, this is not ideal.

```suggestion
For clusters that have root volume availability zones and are running on {rh-openstack} that you upgrade to {product-version}, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in link:https://access.redhat.com/solutions/7013893[OpenShift on OpenStack with Availability Zones: Invalid Compute ServerGroup setup during OpenShift deployment].
```

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (maxwelldb) [reply release_notes/ocp-4-14-release-notes.adoc:1084]:
```suggestion
For clusters that have compute zones configured with at least one zone and are running on {rh-openstack}, which is upgradable to version {product-version}, root volumes must now also be configured with at least one zone. If this configuration change does not occur, a control plane machine set cannot be generated for your cluster. To make the required change, follow the instructions in the related link:https://access.redhat.com/solutions/7024383[OpenShift on OpenStack with compute Availability Zones: Missing rootVolume availability zone].
```

File: release_notes/ocp-4-14-release-notes.adoc (hunk 3 above)
Comment (maxwelldb) [inline release_notes/ocp-4-14-release-notes.adoc:2969]:
```suggestion
* For clusters that have root volume availability zones and are running on {rh-openstack} that you upgrade to {product-version}, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in the link:https://access.redhat.com/solutions/7013893[knowledge base article]. (link:https://issues.redhat.com/browse/OCPBUGS-13300[*OCPBUGS-13300*])
```

**Comment by maxwelldb** [comment]
Small changes made. FYI @kalexand-rh and @EmilienM.

**Comment by itzikb-redhat** [comment]
LGTM

**Review by mandre — COMMENTED** [review]
/lgtm

**Comment by bergerhoffer** [comment]
@maxwelldb Just don't forget to squash your commits before merging

**Comment by sjstout** [comment]
lgtm
